lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -783,6 +783,7 @@ static void on_ziti_event(ziti_context ztx, const ziti_event_t *event) {
                 }
             }
 
+            ziti_tunnel_commit_routes(CMD_CTX.tunnel_ctx);
```

Comment:
commit always called at end of processing service update - I suppose I could just do this myself at the end of my service update handler.  any merit to the idea of TSDK doing all of the add/remove tracking and sending in a list of routes to add and a list of routes to delete?  That way all TSDK users wouldn't need to implement the logic to keep track of each route add and delete